graphql_compiler/tests/snapshot_tests/snapshots/snap_test_orientdb_match_query.py

```diff
@@ -119,7 +119,7 @@
 
 snapshots['OrientDBParameterizedMatchQueryTests::test_between_lowering_with_extra_filters 1'] = frozenset({(frozenset({('name', 'Nazgul__(1_12_14_19)')}), 1), (frozenset({('name', 'Nazgul__((((18_19_2_6)_1_19_6)_(1_12_14_19)_19_5)_0_19_4)')}), 1), (frozenset({('name', 'Nazgul__(((((18_19_2_6)_1_19_6)_15_16_2)_14_8_9)_(11_12_14_16)_17_18)')}), 1), (frozenset({('name', 'Nazgul__((((18_19_2_6)_1_19_6)_(1_12_14_19)_19_5)_17_6_7)')}), 1), (frozenset({('name', 'Nazgul__(((((18_19_2_6)_1_19_6)_15_16_2)_11_14_6)_(11_5_7_8)_(1_12_14_19)_6)')}), 1), (frozenset({('name', 'Nazgul__(((18_19_2_6)_1_19_6)_(1_12_14_19)_19_5)')}), 1)})
 
-snapshots['OrientDBParameterizedMatchQueryTests::test_complex_optional_variables_with_starting_filter 1'] = frozenset({(frozenset({('other_parent_fed_at', '2008-09-23T00:00:00Z'), ('grandparent_fed_at', '2012-07-02T00:00:00Z')}), 2), (frozenset({('other_parent_fed_at', '2009-08-12T00:00:00Z'), ('child_fed_at', '2012-07-02T00:00:00Z'), ('grandparent_fed_at', '2012-07-02T00:00:00Z')}), 1), (frozenset({('child_fed_at', '2012-07-02T00:00:00Z'), ('grandparent_fed_at', '2012-07-02T00:00:00Z')}), 2), (frozenset({('other_parent_fed_at', '2009-08-12T00:00:00Z'), ('grandparent_fed_at', '2012-07-02T00:00:00Z')}), 5), (frozenset({('other_parent_fed_at', '2008-09-23T00:00:00Z'), ('grandparent_fed_at', '2008-09-23T00:00:00Z')}), 2), (frozenset({('other_parent_fed_at', '2003-12-03T00:00:00Z'), ('grandparent_fed_at', '2008-09-23T00:00:00Z')}), 2), (frozenset({('grandparent_fed_at', '2012-07-02T00:00:00Z')}), 5), (frozenset({('other_parent_fed_at', '2003-12-03T00:00:00Z'), ('grandparent_fed_at', '2012-07-02T00:00:00Z')}), 2), (frozenset({('grandparent_fed_at', '2008-09-23T00:00:00Z')}), 5)})
+snapshots['OrientDBParameterizedMatchQueryTests::test_complex_optional_variables_with_starting_filter 1'] = frozenset({(frozenset({('other_parent_fed_at', '2008-09-23T00:00:00'), ('grandparent_fed_at', '2012-07-02T00:00:00')}), 2), (frozenset({('other_parent_fed_at', '2009-08-12T00:00:00'), ('child_fed_at', '2012-07-02T00:00:00'), ('grandparent_fed_at', '2012-07-02T00:00:00')}), 1), (frozenset({('child_fed_at', '2012-07-02T00:00:00'), ('grandparent_fed_at', '2012-07-02T00:00:00')}), 2), (frozenset({('other_parent_fed_at', '2009-08-12T00:00:00'), ('grandparent_fed_at', '2012-07-02T00:00:00')}), 5), (frozenset({('other_parent_fed_at', '2008-09-23T00:00:00'), ('grandparent_fed_at', '2008-09-23T00:00:00')}), 2), (frozenset({('other_parent_fed_at', '2003-12-03T00:00:00'), ('grandparent_fed_at', '2008-09-23T00:00:00')}), 2), (frozenset({('grandparent_fed_at', '2012-07-02T00:00:00')}), 5), (frozenset({('other_parent_fed_at', '2003-12-03T00:00:00'), ('grandparent_fed_at', '2012-07-02T00:00:00')}), 2), (frozenset({('grandparent_fed_at', '2008-09-23T00:00:00')}), 5)})
```

Comment:
The snapshot test change because the returned datetimes are now timezone naive instead of timezone aware because of the change in `STANDARD_DATETIME_FORMAT` which specifies the datetime format of returned datetime values. 

To be extra clear, the returned datetime values aren't empty now. They just lack the "Z" that indicated a UTC timezone.